tests/data/hrrr/test_http_retrieval.py

```diff
@@ -3,7 +3,7 @@
 
 import pandas as pd
 
-from tests.RME_test_case import RMETestCase
+from tests.RME import RMETestCase
```

Comment:
I see this as a readability improvement and another argument for moving the class into the folder.